Firebase/InAppMessaging/Public/FIRInAppMessagingRendering.h

```diff
@@ -217,7 +217,7 @@ NS_SWIFT_NAME(InAppMessagingImageOnlyDisplay)
                      campaignName:(NSString *)campaignName
               renderAsTestMessage:(BOOL)renderAsTestMessage
                       triggerType:(FIRInAppMessagingDisplayTriggerType)triggerType
-                        imageData:(FIRInAppMessagingImageData *)imageData
+                        imageData:(nullable FIRInAppMessagingImageData *)imageData
```

Comment:
FYI: This is a public header change and is a breaking change for Swift users. We should evaluate the best way to proceed with this

Reply:
On second thought, this is *likely* a breaking change but will have to verify. I think if developers are passing in a force-unwrapped variable this could require changes but I'd need to double-check.

Reply:
I will also check and will share my finding when I do.

Reply:
The scenario seems don't cause any compilation errors or warnings:

ObjC - 
@interface MyClass : NSObject
- (void)someMethodWithValue:(nullable NSObject *)obj;
@end

@implementation MyClass
- (void)someMethodWithValue:(nullable NSObject *)obj {}
@end

Swift - 
        let myClass = MyClass()
        let someObj: NSObject? = NSObject()
        myClass.someMethod(withValue: someObj!)

Reply:
As discussed on a separate pull request, these initializers probably shouldn't be public.

Probably not a breaking change in practice, because I can't think of any reason why a developer would actually initialize one of these classes. They just get spun up by the SDK from server responses.